packages/embark-library-manager/src/index.js

```diff
@@ -80,7 +80,7 @@ class LibraryManager {
 
   downloadIfNeeded(packageName, cb) {
     const wantedVersion = this.versions[packageName];
-    let installedVersion = require('../../../../package.json').dependencies[packageName];
+    let installedVersion = this.embark.pkg.dependencies[packageName];
```

Comment:
This package needs access to embark core's dependencies which is why this PR exposes pkg on `this.embark` for now. We probably want to make it an event in the future.